force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -287,11 +356,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
-        List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
+        List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; // DO NOT CHANGE
+        
+        Decimal total = 0;
 
-        return null; // Replace null with the variable you used to store the result
+        for (Decimal amount : moneyInWallet) {
+            total += amount; 
+            if (total > 40.0) {
```

Comment:
Try to use a variable for 40 so it's not a magic number  https://en.wikipedia.org/wiki/Magic_number_(programming)